08LessonEight/in-class-project/css/homework-style.css

```diff
@@ -0,0 +1,139 @@
+header{
+  height:120px;
+  width:100%;
+  text-align: center;
+  font-family: sans-serif;
+  color:grey;
+}
+h1{
+  font-size: 24px;
+  color:grey;
+}
+p{
+  font-size: 10px;
+  color:grey;
+  line-height:20px;
+}
+.image{
+  width:25%;
+  height:295px;
+  background-color: #F7F7F7;
+  padding:10px 10px 10px 10px;
```

Comment:
Yes I learned how last class. I miss s the class that you all went over it . Going to fix tonight 

Sent from my iPhone

> On Jun 5, 2016, at 1:57 PM, Victoria Chuang <notifications@github.com> wrote:
> 
> In 08LessonEight/in-class-project/css/homework-style.css:
> 
> > +  color:grey;
> > +}
> > +h1{
> > +  font-size: 24px;
> > +  color:grey;
> > +}
> > +p{
> > +  font-size: 10px;
> > +  color:grey;
> > +  line-height:20px;
> > +}
> > +.image{
> > +  width:25%;
> > +  height:295px;
> > +  background-color: #F7F7F7;
> > +  padding:10px 10px 10px 10px;
> if the padding is the same in every direction, can you simplify this line?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>